### ISSUE TITLE
[Bug]: All apps crashes at launch under macOS Sonoma 14.2.1

### ISSUE THREAD

**Issue by dzy1997** [PlayCover/PlayCover#1324]
### Describe the bug

After I update my macOS to Sonoma 14.2.1, all my apps in Playcover crashes at launch. 
I tried to read crash logs. Surprisingly, different apps seem to have different "Termination Reason", we have "Namespace DYLD, Code 4 Symbol missing", "Namespace SIGNAL, Code 5 Trace/BPT trap: 5" and "Namespace SIGNAL, Code 11 Segmentation fault: 11". All apps worked properly before the system update.

### Steps to reproduce

Just double click to launch any app

### Expected behaviour

The app should launch (but it does not)

### Crash log

```shell
-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               Yuanshen [52619]
Path:                  /Users/USER/Library/Containers/io.playcover.PlayCover/原神.app/Yuanshen
Identifier:            com.miHoYo.Yuanshen
Version:               4.3.0 (716)
Code Type:             ARM-64 (Native)
Parent Process:        launchd [1]
User ID:               501

Date/Time:             2024-01-15 20:38:28.3733 +0800
OS Version:            macOS 14.2.1 (23C71)
Report Version:        12
Anonymous UUID:        83C2D39A-F9DD-A854-ACD5-5B94E3DC703D

Sleep/Wake UUID:       916CC1BE-43C9-4EDB-802E-8BE96231BBE7

Time Awake Since Boot: 33000 seconds
Time Since Wake:       11249 seconds

System Integrity Protection: enabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_CRASH (SIGTRAP)
Exception Codes:       0x0000000000000000, 0x0000000000000000

Termination Reason:    Namespace SIGNAL, Code 5 Trace/BPT trap: 5
Terminating Process:   Yuanshen [52619]

Thread 0 Crashed::  Dispatch queue: com.apple.main-thread
0   libsystem_kernel.dylib        	       0x187b85874 mach_msg2_trap + 8
1   libsystem_kernel.dylib        	       0x187b97cf0 mach_msg2_internal + 80
2   libsystem_kernel.dylib        	       0x187b8e4b0 mach_msg_overwrite + 476
3   libsystem_kernel.dylib        	       0x187b85bf8 mach_msg + 24
4   SkyLight                      	       0x18d39eaa4 SLSConnectionGetHeldWindowRights + 176
5   AppKit                        	       0x18c148a44 -[NSCGSWindow initWithWindowID:] + 108
6   AppKit                        	       0x18b49f974 +[NSCGSWindow(NSCGSWindowCreation) windowWithWindowID:] + 96
7   AppKit                        	       0x18bfe3be0 -[_NSCGSWindowOrdering _windowsWithOptions:onSpaces:xorSpacesMatching:forConnectionID:] + 460
8   AppKit                        	       0x18b47983c -[NSApplication enumerateWindowsWithOptions:usingBlock:] + 324
9   AppKit                        	       0x18b4796b8 -[NSApplication _findWindowWithOptions:passingTest:] + 160
10  AppKit                        	       0x18b66feb8 -[NSApplication _bestKeyWindowCandidateOnScreen:] + 192
11  AppKit                        	       0x18b486114 -[NSApplication _sendFinishLaunchingNotification] + 400
12  AppKit                        	       0x18b484578 -[NSApplication(NSAppleEventHandling) _handleAEOpenEvent:] + 504
13  AppKit                        	       0x18b484174 -[NSApplication(NSAppleEventHandling) _handleCoreEvent:withReplyEvent:] + 492
14  Foundation                    	       0x188d82414 -[NSAppleEventManager dispatchRawAppleEvent:withRawReply:handlerRefCon:] + 316
15  Foundation                    	       0x188d82208 _NSAppleEventManagerGenericHandler + 80
16  AE                            	       0x18ebf8dc0 0x18ebed000 + 48576
17  AE                            	       0x18ebf86e8 0x18ebed000 + 46824
18  AE                            	       0x18ebf1cf8 aeProcessAppleEvent + 488
19  HIToolbox                     	       0x1922602d4 AEProcessAppleEvent + 68
20  AppKit                        	       0x18b47ebb0 _DPSNextEvent + 1440
21  AppKit                        	       0x18bc58980 -[NSApplication(NSEventRouting) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 716
22  AppKit                        	       0x18b471d50 -[NSApplication run] + 476
23  AppKit                        	       0x18b449014 NSApplicationMain + 880
24  AppKit                        	       0x18b69c3a4 _NSApplicationMainWithInfoDictionary + 24
25  UIKitMacHelper                	       0x1a09196a8 UINSApplicationMain + 972
26  UIKitCore                     	       0x1b64e4cfc UIApplicationMain + 148
27  Yuanshen                      	       0x102175098 0x102170000 + 20632
28  dyld                          	       0x1878450e0 start + 2360

Thread 1:
0   libsystem_pthread.dylib       	       0x187bc0e28 start_wqthread + 0

Thread 2:
0   libsystem_pthread.dylib       	       0x187bc0e28 start_wqthread + 0

Thread 3:
0   libsystem_pthread.dylib       	       0x187bc0e28 start_wqthread + 0

Thread 4:: ShaderLoadManagerT
0   libsystem_kernel.dylib        	       0x187b85808 semaphore_timedwait_trap + 8
1   libdispatch.dylib             	       0x187a14f28 _dispatch_sema4_timedwait + 64
2   libdispatch.dylib             	       0x187a15524 _dispatch_semaphore_wait_slow + 76
3   Yuanshen                      	       0x10a651a08 0x102170000 + 139336200
4   Yuanshen                      	       0x10a64f324 0x102170000 + 139326244
5   Yuanshen                      	       0x10a632430 0x102170000 + 139207728
6   libsystem_pthread.dylib       	       0x187bc6034 _pthread_start + 136
7   libsystem_pthread.dylib       	       0x187bc0e3c thread_start + 8

Thread 5:: com.apple.uikit.eventfetch-thread
0   libsystem_kernel.dylib        	       0x187b85874 mach_msg2_trap + 8
1   libsystem_kernel.dylib        	       0x187b97cf0 mach_msg2_internal + 80
2   libsystem_kernel.dylib        	       0x187b8e4b0 mach_msg_overwrite + 476
3   libsystem_kernel.dylib        	       0x187b85bf8 mach_msg + 24
4   CoreFoundation                	       0x187ca3bf4 __CFRunLoopServiceMachPort + 160
5   CoreFoundation                	       0x187ca24bc __CFRunLoopRun + 1208
6   CoreFoundation                	       0x187ca19ac CFRunLoopRunSpecific + 608
7   Foundation                    	       0x188daba38 -[NSRunLoop(NSRunLoop) runMode:beforeDate:] + 212
8   Foundation                    	       0x188e203b0 -[NSRunLoop(NSRunLoop) runUntilDate:] + 100
9   UIKitCore                     	       0x1b64e60c8 -[UIEventFetcher threadMain] + 104
10  Foundation                    	       0x188da4fb4 __NSThread__start__ + 716
11  libsystem_pthread.dylib       	       0x187bc6034 _pthread_start + 136
12  libsystem_pthread.dylib       	       0x187bc0e3c thread_start + 8


Thread 0 crashed with ARM Thread State (64-bit):
    x0: 0x0000000010004005   x1: 0x0000000400304003   x2: 0x0000002400001513   x3: 0x00001d0300005403
    x4: 0x0000732400000000   x5: 0x00001d0300000000   x6: 0x0000000000000038   x7: 0x0000000000000000
    x8: 0xfffffffffffffbbf   x9: 0x0000000000001d03  x10: 0x0000000000007324  x11: 0x0000000000000001
   x12: 0x0000000000001d03  x13: 0x0000000126f2a320  x14: 0x01000001de3bcdad  x15: 0x00000001de3bcda8
   x16: 0xffffffffffffffd1  x17: 0x00000001e74135d0  x18: 0x0000000000000000  x19: 0x0000000000000000
   x20: 0x0000000000000038  x21: 0x00001d0300000000  x22: 0x0000732400000000  x23: 0x00001d0300005403
   x24: 0x000000016dc8e750  x25: 0x0000002400001513  x26: 0x0000000400304003  x27: 0x0000000400304003
   x28: 0x0000000000304003   fp: 0x000000016dc8e600   lr: 0x0000000187b97cf0
    sp: 0x000000016dc8e5b0   pc: 0x0000000187b85874 cpsr: 0x20001000
   far: 0x0000000000000000  esr: 0x56000080  Address size fault

Binary Images:
       0x110af4000 -        0x110af7fff io.playcover.AKInterface (1.0) <040c80cc-48b9-323d-afc8-85d7903eabb2> /Users/USER/Library/Containers/io.playcover.PlayCover/原神.app/PlugIns/AKInterface.bundle/Contents/MacOS/AKInterface
       0x110944000 -        0x11094ffff libobjc-trampolines.dylib (*) <7778e0d7-361a-378d-9438-3b2bb48c2154> /usr/lib/libobjc-trampolines.dylib
       0x110e38000 -        0x110fb7fff io.playcover.PlayTools (1.1.7) <123137f0-4d0e-3516-8842-568170da73e7> /Users/USER/Library/Frameworks/PlayTools.framework/PlayTools
       0x102170000 -        0x10de7ffff com.miHoYo.Yuanshen (4.3.0) <ca5935bd-9bd4-35b9-b74a-5e999f1fae80> /Users/USER/Library/Containers/io.playcover.PlayCover/原神.app/Yuanshen
       0x187b84000 -        0x187bbefff libsystem_kernel.dylib (*) <ca94fc21-bc40-3b43-b65d-b87ece9e1d48> /usr/lib/system/libsystem_kernel.dylib
       0x18d2cf000 -        0x18d764fff com.apple.SkyLight (1.600.0) <67c718b4-52dc-3b25-8e7a-12ea2b414369> /System/Library/PrivateFrameworks/SkyLight.framework/Versions/A/SkyLight
       0x18b445000 -        0x18c75ffff com.apple.AppKit (6.9) <f3527312-e426-3f7c-b77b-2bf49d1b7c04> /System/Library/Frameworks/AppKit.framework/Versions/C/AppKit
       0x188d51000 -        0x189998fff com.apple.Foundation (6.9) <9558b1eb-dda3-3fda-88a5-e785ecdfcd30> /System/Library/Frameworks/Foundation.framework/Versions/C/Foundation
       0x18ebed000 -        0x18ec60ff7 com.apple.AE (944) <3beca17e-6610-32dd-80ca-805b81f0c933> /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/AE.framework/Versions/A/AE
       0x192220000 -        0x1924dffff com.apple.HIToolbox (2.1.1) <e2187dfe-1fb4-3b47-874e-a0a6291e51b2> /System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/HIToolbox.framework/Versions/A/HIToolbox
       0x1a0915000 -        0x1a0a27fff com.apple.UIKitMacHelper (1.0) <f03f3f95-a137-39e2-b894-c76dc6e8a1e5> /System/Library/PrivateFrameworks/UIKitMacHelper.framework/Versions/A/UIKitMacHelper
       0x1b64e1000 -        0x1b7e2afff com.apple.UIKitCore (1.0) <2aebcc42-a1aa-3535-be53-1da03d4d2ce3> /System/iOSSupport/System/Library/PrivateFrameworks/UIKitCore.framework/Versions/A/UIKitCore
       0x18783f000 -        0x1878d3347 dyld (*) <324e4ad9-e01f-3183-b09f-3e20b326643a> /usr/lib/dyld
               0x0 - 0xffffffffffffffff ??? (*) <00000000-0000-0000-0000-000000000000> ???
       0x187bbf000 -        0x187bcbff3 libsystem_pthread.dylib (*) <a7d94c96-7b1f-3229-9bea-048d037c3292> /usr/lib/system/libsystem_pthread.dylib
       0x187a11000 -        0x187a57fff libdispatch.dylib (*) <a53d555d-f748-3010-83fe-385c660a81bd> /usr/lib/system/libdispatch.dylib
       0x187c26000 -        0x1880fdfff com.apple.CoreFoundation (6.9) <47e4ec09-8f6e-30a8-99d0-34024d4f8122> /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation

External Modification Summary:
  Calls made by other processes targeting this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by all processes on this machine:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0

VM Region Summary:
ReadOnly portion of Libraries: Total=1.9G resident=0K(0%) swapped_out_or_unallocated=1.9G(100%)
Writable regions: Total=1.5G written=0K(0%) resident=0K(0%) swapped_out=0K(0%) unallocated=1.5G(100%)

                                VIRTUAL   REGION 
REGION TYPE                        SIZE    COUNT (non-coalesced) 
===========                     =======  ======= 
Activity Tracing                   256K        1 
ColorSync                          544K       26 
CoreAnimation                      320K       20 
CoreGraphics                        48K        3 
Foundation                          16K        1 
Kernel Alloc Once                   32K        1 
MALLOC                             1.5G       48 
MALLOC guard page                  192K       12 
SQLite page cache                  256K        2 
STACK GUARD                       56.1M        6 
Stack                             10.9M        7 
Stack Guard                         16K        1 
VM_ALLOCATE                        112K        7 
__AUTH                            3615K      539 
__AUTH_CONST                      37.4M      775 
__CTF                               824        1 
__DATA                            50.2M      766 
__DATA_CONST                      40.5M      786 
__DATA_DIRTY                      2665K      311 
__FONT_DATA                          4K        1 
__LINKEDIT                       900.2M        5 
__OBJC_RO                         71.1M        1 
__OBJC_RW                         2168K        1 
__TEXT                           997.0M      808 
dyld private memory                272K        2 
mapped file                      297.3M      240 
shared memory                     1360K       13 
===========                     =======  ======= 
TOTAL                              3.9G     4384 



-----------
Full Report
-----------

{"app_name":"Yuanshen","timestamp":"2024-01-15 20:38:28.00 +0800","app_version":"4.3.0","slice_uuid":"ca5935bd-9bd4-35b9-b74a-5e999f1fae80","build_version":"716","platform":6,"bundleID":"com.miHoYo.Yuanshen","share_with_app_devs":1,"is_first_party":0,"bug_type":"309","os_version":"macOS 14.2.1 (23C71)","roots_installed":0,"name":"Yuanshen","incident_id":"44C9CBFA-D239-4BD3-BE0E-BABCDCBDEF08"}
{
  "uptime" : 33000,
  "procRole" : "Foreground",
  "version" : 2,
  "userID" : 501,
  "deployVersion" : 210,
  "modelCode" : "Mac14,10",
  "coalitionID" : 18644,
  "osVersion" : {
    "train" : "macOS 14.2.1",
    "build" : "23C71",
    "releaseType" : "User"
  },
  "captureTime" : "2024-01-15 20:38:28.3733 +0800",
  "codeSigningMonitor" : 1,
  "incident" : "44C9CBFA-D239-4BD3-BE0E-BABCDCBDEF08",
  "pid" : 52619,
  "translated" : false,
  "cpuType" : "ARM-64",
  "roots_installed" : 0,
  "bug_type" : "309",
  "procLaunch" : "2024-01-15 20:38:28.0693 +0800",
  "procStartAbsTime" : 799083012167,
  "procExitAbsTime" : 799090279314,
  "procName" : "Yuanshen",
  "procPath" : "\/Users\/USER\/Library\/Containers\/io.playcover.PlayCover\/原神.app\/Yuanshen",
  "bundleInfo" : {"CFBundleShortVersionString":"4.3.0","CFBundleVersion":"716","CFBundleIdentifier":"com.miHoYo.Yuanshen","DTAppStoreToolsBuild":"15C5065a"},
  "storeInfo" : {"deviceIdentifierForVendor":"D9A86444-FDCF-57E4-821C-000EEBCE7187","thirdParty":true},
  "parentProc" : "launchd",
  "parentPid" : 1,
  "coalitionName" : "com.miHoYo.Yuanshen",
  "crashReporterKey" : "83C2D39A-F9DD-A854-ACD5-5B94E3DC703D",
  "codeSigningID" : "com.miHoYo.Yuanshen",
  "codeSigningTeamID" : "",
  "codeSigningFlags" : 570425857,
  "codeSigningValidationCategory" : 10,
  "codeSigningTrustLevel" : 4294967295,
  "instructionByteStream" : {"beforePC":"ARAA1MADX9aQBYCSARAA1MADX9awBYCSARAA1MADX9bQBYCSARAA1A==","atPC":"wANf1vAFgJIBEADUwANf1hAGgJIBEADUwANf1jAGgJIBEADUwANf1g=="},
  "wakeTime" : 11249,
  "sleepWakeUUID" : "916CC1BE-43C9-4EDB-802E-8BE96231BBE7",
  "sip" : "enabled",
  "exception" : {"codes":"0x0000000000000000, 0x0000000000000000","rawCodes":[0,0],"type":"EXC_CRASH","signal":"SIGTRAP"},
  "termination" : {"flags":0,"code":5,"namespace":"SIGNAL","indicator":"Trace\/BPT trap: 5","byProc":"Yuanshen","byPid":52619},
  "os_fault" : {"process":"Yuanshen"},
  "extMods" : {"caller":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"system":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"targeted":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"warnings":0},
  "faultingThread" : 0,
  "threads" : [{"triggered":true,"id":707510,"threadState":{"x":[{"value":268451845},{"value":17183031299},{"value":154618828051},{"value":31898722128899},{"value":126598456016896},{"value":31898722107392},{"value":56},{"value":0},{"value":18446744073709550527},{"value":7427},{"value":29476},{"value":1},{"value":7427},{"value":4948402976},{"value":72057602061356461,"symbolLocation":72057594037927941,"symbol":"OBJC_CLASS_$___NSCFType"},{"value":8023428520,"symbolLocation":0,"symbol":"OBJC_CLASS_$___NSCFType"},{"value":18446744073709551569},{"value":8174777808},{"value":0},{"value":0},{"value":56},{"value":31898722107392},{"value":126598456016896},{"value":31898722128899},{"value":6136850256},{"value":154618828051},{"value":17183031299},{"value":17183031299},{"value":3162115}],"flavor":"ARM_THREAD_STATE64","lr":{"value":6572047600},"cpsr":{"value":536875008},"fp":{"value":6136849920},"sp":{"value":6136849840},"esr":{"value":1442840704,"description":" Address size fault"},"pc":{"value":6571972724,"matchesCrashFrame":1},"far":{"value":0}},"queue":"com.apple.main-thread","frames":[{"imageOffset":6260,"symbol":"mach_msg2_trap","symbolLocation":8,"imageIndex":4},{"imageOffset":81136,"symbol":"mach_msg2_internal","symbolLocation":80,"imageIndex":4},{"imageOffset":42160,"symbol":"mach_msg_overwrite","symbolLocation":476,"imageIndex":4},{"imageOffset":7160,"symbol":"mach_msg","symbolLocation":24,"imageIndex":4},{"imageOffset":850596,"symbol":"SLSConnectionGetHeldWindowRights","symbolLocation":176,"imageIndex":5},{"imageOffset":13646404,"symbol":"-[NSCGSWindow initWithWindowID:]","symbolLocation":108,"imageIndex":6},{"imageOffset":371060,"symbol":"+[NSCGSWindow(NSCGSWindowCreation) windowWithWindowID:]","symbolLocation":96,"imageIndex":6},{"imageOffset":12184544,"symbol":"-[_NSCGSWindowOrdering _windowsWithOptions:onSpaces:xorSpacesMatching:forConnectionID:]","symbolLocation":460,"imageIndex":6},{"imageOffset":215100,"symbol":"-[NSApplication enumerateWindowsWithOptions:usingBlock:]","symbolLocation":324,"imageIndex":6},{"imageOffset":214712,"symbol":"-[NSApplication _findWindowWithOptions:passingTest:]","symbolLocation":160,"imageIndex":6},{"imageOffset":2272952,"symbol":"-[NSApplication _bestKeyWindowCandidateOnScreen:]","symbolLocation":192,"imageIndex":6},{"imageOffset":266516,"symbol":"-[NSApplication _sendFinishLaunchingNotification]","symbolLocation":400,"imageIndex":6},{"imageOffset":259448,"symbol":"-[NSApplication(NSAppleEventHandling) _handleAEOpenEvent:]","symbolLocation":504,"imageIndex":6},{"imageOffset":258420,"symbol":"-[NSApplication(NSAppleEventHandling) _handleCoreEvent:withReplyEvent:]","symbolLocation":492,"imageIndex":6},{"imageOffset":201748,"symbol":"-[NSAppleEventManager dispatchRawAppleEvent:withRawReply:handlerRefCon:]","symbolLocation":316,"imageIndex":7},{"imageOffset":201224,"symbol":"_NSAppleEventManagerGenericHandler","symbolLocation":80,"imageIndex":7},{"imageOffset":48576,"imageIndex":8},{"imageOffset":46824,"imageIndex":8},{"imageOffset":19704,"symbol":"aeProcessAppleEvent","symbolLocation":488,"imageIndex":8},{"imageOffset":262868,"symbol":"AEProcessAppleEvent","symbolLocation":68,"imageIndex":9},{"imageOffset":236464,"symbol":"_DPSNextEvent","symbolLocation":1440,"imageIndex":6},{"imageOffset":8468864,"symbol":"-[NSApplication(NSEventRouting) _nextEventMatchingEventMask:untilDate:inMode:dequeue:]","symbolLocation":716,"imageIndex":6},{"imageOffset":183632,"symbol":"-[NSApplication run]","symbolLocation":476,"imageIndex":6},{"imageOffset":16404,"symbol":"NSApplicationMain","symbolLocation":880,"imageIndex":6},{"imageOffset":2454436,"symbol":"_NSApplicationMainWithInfoDictionary","symbolLocation":24,"imageIndex":6},{"imageOffset":18088,"symbol":"UINSApplicationMain","symbolLocation":972,"imageIndex":10},{"imageOffset":15612,"symbol":"UIApplicationMain","symbolLocation":148,"imageIndex":11},{"imageOffset":20632,"imageIndex":3},{"imageOffset":24800,"symbol":"start","symbolLocation":2360,"imageIndex":12}]},{"id":707529,"frames":[{"imageOffset":7720,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":14}],"threadState":{"x":[{"value":6137982976},{"value":4867},{"value":6137446400},{"value":0},{"value":409604},{"value":18446744073709551615},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0}],"flavor":"ARM_THREAD_STATE64","lr":{"value":0},"cpsr":{"value":4096},"fp":{"value":0},"sp":{"value":6137982976},"esr":{"value":1442840704,"description":" Address size fault"},"pc":{"value":6572215848},"far":{"value":0}}},{"id":707531,"frames":[{"imageOffset":7720,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":14}],"threadState":{"x":[{"value":6138556416},{"value":16391},{"value":6138019840},{"value":0},{"value":409604},{"value":18446744073709551615},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0}],"flavor":"ARM_THREAD_STATE64","lr":{"value":0},"cpsr":{"value":4096},"fp":{"value":0},"sp":{"value":6138556416},"esr":{"value":1442840704,"description":" Address size fault"},"pc":{"value":6572215848},"far":{"value":0}}},{"id":707533,"frames":[{"imageOffset":7720,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":14}],"threadState":{"x":[{"value":6139129856},{"value":18947},{"value":6138593280},{"value":0},{"value":409604},{"value":18446744073709551615},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0}],"flavor":"ARM_THREAD_STATE64","lr":{"value":0},"cpsr":{"value":4096},"fp":{"value":0},"sp":{"value":6139129856},"esr":{"value":1442840704,"description":" Address size fault"},"pc":{"value":6572215848},"far":{"value":0}}},{"id":707537,"name":"ShaderLoadManagerT","threadState":{"x":[{"value":14},{"value":42949492571373568},{"value":9999958},{"value":68719460488},{"value":105553134374272},{"value":3360},{"value":48},{"value":0},{"value":9999958},{"value":12297829382473034411},{"value":13835058055282163714},{"value":160000},{"value":1099511627776},{"value":0},{"value":0},{"value":0},{"value":18446744073709551578},{"value":8089001952},{"value":0},{"value":799090420109},{"value":105553134374240},{"value":1000000000},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0}],"flavor":"ARM_THREAD_STATE64","lr":{"value":6570463016},"cpsr":{"value":2147487744},"fp":{"value":6139440864},"sp":{"value":6139440832},"esr":{"value":1442840704,"description":" Address size fault"},"pc":{"value":6571972616},"far":{"value":0}},"frames":[{"imageOffset":6152,"symbol":"semaphore_timedwait_trap","symbolLocation":8,"imageIndex":4},{"imageOffset":16168,"symbol":"_dispatch_sema4_timedwait","symbolLocation":64,"imageIndex":15},{"imageOffset":17700,"symbol":"_dispatch_semaphore_wait_slow","symbolLocation":76,"imageIndex":15},{"imageOffset":139336200,"imageIndex":3},{"imageOffset":139326244,"imageIndex":3},{"imageOffset":139207728,"imageIndex":3},{"imageOffset":28724,"symbol":"_pthread_start","symbolLocation":136,"imageIndex":14},{"imageOffset":7740,"symbol":"thread_start","symbolLocation":8,"imageIndex":14}]},{"id":707544,"name":"com.apple.uikit.eventfetch-thread","threadState":{"x":[{"value":268451845},{"value":21592279046},{"value":8589934592},{"value":111063559307264},{"value":0},{"value":111063559307264},{"value":2},{"value":4294967295},{"value":18446744073709550527},{"value":25859},{"value":0},{"value":1},{"value":25859},{"value":242905},{"value":0},{"value":0},{"value":18446744073709551569},{"value":8174777808},{"value":0},{"value":4294967295},{"value":2},{"value":111063559307264},{"value":0},{"value":111063559307264},{"value":6140009864},{"value":8589934592},{"value":21592279046},{"value":21592279046},{"value":4412409862}],"flavor":"ARM_THREAD_STATE64","lr":{"value":6572047600},"cpsr":{"value":4096},"fp":{"value":6140009712},"sp":{"value":6140009632},"esr":{"value":1442840704,"description":" Address size fault"},"pc":{"value":6571972724},"far":{"value":0}},"frames":[{"imageOffset":6260,"symbol":"mach_msg2_trap","symbolLocation":8,"imageIndex":4},{"imageOffset":81136,"symbol":"mach_msg2_internal","symbolLocation":80,"imageIndex":4},{"imageOffset":42160,"symbol":"mach_msg_overwrite","symbolLocation":476,"imageIndex":4},{"imageOffset":7160,"symbol":"mach_msg","symbolLocation":24,"imageIndex":4},{"imageOffset":515060,"symbol":"__CFRunLoopServiceMachPort","symbolLocation":160,"imageIndex":16},{"imageOffset":509116,"symbol":"__CFRunLoopRun","symbolLocation":1208,"imageIndex":16},{"imageOffset":506284,"symbol":"CFRunLoopRunSpecific","symbolLocation":608,"imageIndex":16},{"imageOffset":371256,"symbol":"-[NSRunLoop(NSRunLoop) runMode:beforeDate:]","symbolLocation":212,"imageIndex":7},{"imageOffset":848816,"symbol":"-[NSRunLoop(NSRunLoop) runUntilDate:]","symbolLocation":100,"imageIndex":7},{"imageOffset":20680,"symbol":"-[UIEventFetcher threadMain]","symbolLocation":104,"imageIndex":11},{"imageOffset":343988,"symbol":"__NSThread__start__","symbolLocation":716,"imageIndex":7},{"imageOffset":28724,"symbol":"_pthread_start","symbolLocation":136,"imageIndex":14},{"imageOffset":7740,"symbol":"thread_start","symbolLocation":8,"imageIndex":14}]}],
  "usedImages" : [
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4574887936,
    "CFBundleShortVersionString" : "1.0",
    "CFBundleIdentifier" : "io.playcover.AKInterface",
    "size" : 16384,
    "uuid" : "040c80cc-48b9-323d-afc8-85d7903eabb2",
    "path" : "\/Users\/USER\/Library\/Containers\/io.playcover.PlayCover\/原神.app\/PlugIns\/AKInterface.bundle\/Contents\/MacOS\/AKInterface",
    "name" : "AKInterface",
    "CFBundleVersion" : "1"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 4573118464,
    "size" : 49152,
    "uuid" : "7778e0d7-361a-378d-9438-3b2bb48c2154",
    "path" : "\/usr\/lib\/libobjc-trampolines.dylib",
    "name" : "libobjc-trampolines.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4578312192,
    "CFBundleShortVersionString" : "1.1.7",
    "CFBundleIdentifier" : "io.playcover.PlayTools",
    "size" : 1572864,
    "uuid" : "123137f0-4d0e-3516-8842-568170da73e7",
    "path" : "\/Users\/USER\/Library\/Frameworks\/PlayTools.framework\/PlayTools",
    "name" : "PlayTools",
    "CFBundleVersion" : "1"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4330029056,
    "CFBundleShortVersionString" : "4.3.0",
    "CFBundleIdentifier" : "com.miHoYo.Yuanshen",
    "size" : 198246400,
    "uuid" : "ca5935bd-9bd4-35b9-b74a-5e999f1fae80",
    "path" : "\/Users\/USER\/Library\/Containers\/io.playcover.PlayCover\/原神.app\/Yuanshen",
    "name" : "Yuanshen",
    "CFBundleVersion" : "716"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6571966464,
    "size" : 241664,
    "uuid" : "ca94fc21-bc40-3b43-b65d-b87ece9e1d48",
    "path" : "\/usr\/lib\/system\/libsystem_kernel.dylib",
    "name" : "libsystem_kernel.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6663499776,
    "CFBundleShortVersionString" : "1.600.0",
    "CFBundleIdentifier" : "com.apple.SkyLight",
    "size" : 4808704,
    "uuid" : "67c718b4-52dc-3b25-8e7a-12ea2b414369",
    "path" : "\/System\/Library\/PrivateFrameworks\/SkyLight.framework\/Versions\/A\/SkyLight",
    "name" : "SkyLight"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6631477248,
    "CFBundleShortVersionString" : "6.9",
    "CFBundleIdentifier" : "com.apple.AppKit",
    "size" : 20033536,
    "uuid" : "f3527312-e426-3f7c-b77b-2bf49d1b7c04",
    "path" : "\/System\/Library\/Frameworks\/AppKit.framework\/Versions\/C\/AppKit",
    "name" : "AppKit",
    "CFBundleVersion" : "2487.30.108"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6590631936,
    "CFBundleShortVersionString" : "6.9",
    "CFBundleIdentifier" : "com.apple.Foundation",
    "size" : 12877824,
    "uuid" : "9558b1eb-dda3-3fda-88a5-e785ecdfcd30",
    "path" : "\/System\/Library\/Frameworks\/Foundation.framework\/Versions\/C\/Foundation",
    "name" : "Foundation",
    "CFBundleVersion" : "2202"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6689837056,
    "CFBundleShortVersionString" : "944",
    "CFBundleIdentifier" : "com.apple.AE",
    "size" : 475128,
    "uuid" : "3beca17e-6610-32dd-80ca-805b81f0c933",
    "path" : "\/System\/Library\/Frameworks\/CoreServices.framework\/Versions\/A\/Frameworks\/AE.framework\/Versions\/A\/AE",
    "name" : "AE",
    "CFBundleVersion" : "944"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6746669056,
    "CFBundleShortVersionString" : "2.1.1",
    "CFBundleIdentifier" : "com.apple.HIToolbox",
    "size" : 2883584,
    "uuid" : "e2187dfe-1fb4-3b47-874e-a0a6291e51b2",
    "path" : "\/System\/Library\/Frameworks\/Carbon.framework\/Versions\/A\/Frameworks\/HIToolbox.framework\/Versions\/A\/HIToolbox",
    "name" : "HIToolbox"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6988845056,
    "CFBundleShortVersionString" : "1.0",
    "CFBundleIdentifier" : "com.apple.UIKitMacHelper",
    "size" : 1126400,
    "uuid" : "f03f3f95-a137-39e2-b894-c76dc6e8a1e5",
    "path" : "\/System\/Library\/PrivateFrameworks\/UIKitMacHelper.framework\/Versions\/A\/UIKitMacHelper",
    "name" : "UIKitMacHelper",
    "CFBundleVersion" : "7209.1.401"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 7353536512,
    "CFBundleShortVersionString" : "1.0",
    "CFBundleIdentifier" : "com.apple.UIKitCore",
    "size" : 26517504,
    "uuid" : "2aebcc42-a1aa-3535-be53-1da03d4d2ce3",
    "path" : "\/System\/iOSSupport\/System\/Library\/PrivateFrameworks\/UIKitCore.framework\/Versions\/A\/UIKitCore",
    "name" : "UIKitCore",
    "CFBundleVersion" : "7209.1.401"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6568538112,
    "size" : 607048,
    "uuid" : "324e4ad9-e01f-3183-b09f-3e20b326643a",
    "path" : "\/usr\/lib\/dyld",
    "name" : "dyld"
  },
  {
    "size" : 0,
    "source" : "A",
    "base" : 0,
    "uuid" : "00000000-0000-0000-0000-000000000000"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6572208128,
    "size" : 53236,
    "uuid" : "a7d94c96-7b1f-3229-9bea-048d037c3292",
    "path" : "\/usr\/lib\/system\/libsystem_pthread.dylib",
    "name" : "libsystem_pthread.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6570446848,
    "size" : 290816,
    "uuid" : "a53d555d-f748-3010-83fe-385c660a81bd",
    "path" : "\/usr\/lib\/system\/libdispatch.dylib",
    "name" : "libdispatch.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6572630016,
    "CFBundleShortVersionString" : "6.9",
    "CFBundleIdentifier" : "com.apple.CoreFoundation",
    "size" : 5079040,
    "uuid" : "47e4ec09-8f6e-30a8-99d0-34024d4f8122",
    "path" : "\/System\/Library\/Frameworks\/CoreFoundation.framework\/Versions\/A\/CoreFoundation",
    "name" : "CoreFoundation",
    "CFBundleVersion" : "2202"
  }
],
  "sharedCache" : {
  "base" : 6567788544,
  "size" : 4061216768,
  "uuid" : "f9ddd844-7f3f-34bd-be29-f0c72d5e5449"
},
  "vmSummary" : "ReadOnly portion of Libraries: Total=1.9G resident=0K(0%) swapped_out_or_unallocated=1.9G(100%)\nWritable regions: Total=1.5G written=0K(0%) resident=0K(0%) swapped_out=0K(0%) unallocated=1.5G(100%)\n\n                                VIRTUAL   REGION \nREGION TYPE                        SIZE    COUNT (non-coalesced) \n===========                     =======  ======= \nActivity Tracing                   256K        1 \nColorSync                          544K       26 \nCoreAnimation                      320K       20 \nCoreGraphics                        48K        3 \nFoundation                          16K        1 \nKernel Alloc Once                   32K        1 \nMALLOC                             1.5G       48 \nMALLOC guard page                  192K       12 \nSQLite page cache                  256K        2 \nSTACK GUARD                       56.1M        6 \nStack                             10.9M        7 \nStack Guard                         16K        1 \nVM_ALLOCATE                        112K        7 \n__AUTH                            3615K      539 \n__AUTH_CONST                      37.4M      775 \n__CTF                               824        1 \n__DATA                            50.2M      766 \n__DATA_CONST                      40.5M      786 \n__DATA_DIRTY                      2665K      311 \n__FONT_DATA                          4K        1 \n__LINKEDIT                       900.2M        5 \n__OBJC_RO                         71.1M        1 \n__OBJC_RW                         2168K        1 \n__TEXT                           997.0M      808 \ndyld private memory                272K        2 \nmapped file                      297.3M      240 \nshared memory                     1360K       13 \n===========                     =======  ======= \nTOTAL                              3.9G     4384 \n",
  "legacyInfo" : {
  "threadTriggered" : {
    "queue" : "com.apple.main-thread"
  }
},
  "logWritingSignature" : "54ff8e7a41a9f6d419714e2745d30ad928f90bcf",
  "trialInfo" : {
  "rollouts" : [
    {
      "rolloutId" : "639124e81d92412bfb4880b3",
      "factorPackIds" : {

      },
      "deploymentId" : 240000012
    },
    {
      "rolloutId" : "6425c75e4327780c10cc4252",
      "factorPackIds" : {
        "SIRI_HOME_AUTOMATION_INTENT_SELECTION_CACHE" : "642600a457e7664b1698eb32"
      },
      "deploymentId" : 240000004
    }
  ],
  "experiments" : [

  ]
}
}

Model: Mac14,10, BootROM 10151.61.4, proc 12:8:4 processors, 16 GB, SMC 
Graphics: Apple M2 Pro, Apple M2 Pro, Built-In
Display: Color LCD, 3456 x 2234 Retina, Main, MirrorOff, Online
Memory Module: LPDDR5, Micron
AirPort: spairport_wireless_card_type_wifi (0x14E4, 0x4388), wl0: Nov  3 2023 04:54:29 version 23.10.765.17.41.51.127 FWID 01-0717abf7
AirPort: 
Bluetooth: Version (null), 0 services, 0 devices, 0 incoming serial ports
Network Service: Wi-Fi, AirPort, en0
USB Device: USB31Bus
USB Device: USB31Bus
USB Device: USB31Bus
Thunderbolt Bus: MacBook Pro, Apple Inc.
Thunderbolt Bus: MacBook Pro, Apple Inc.
Thunderbolt Bus: MacBook Pro, Apple Inc.
```

### What version of PlayCover are you using?

2.0.5

### What version of macOS are you using?

Sonoma (macOS 14)

### Issue Language

- [x] Yes my issue is written in English

**Comment by ShuangLau** [comment]
+1 has met the same issue. Termination Issue being: `"Namespace SIGNAL, Code 5 Trace/BPT trap: 5" and `

**Comment by s7v3n7** [comment]
Same here, No Apps can launch on V2.0.5 and V3.0.2 beta does not launch at all or rather has the malicious message lol.

**Comment by Moonshee5288** [comment]
Me too, this really bugs me because many of the games i want to play can't be accessed anymore, and i can't play with friends.

**Comment by MEMORIEmusic** [comment]
> Same here, No Apps can launch on V2.0.5 and V3.0.2 beta does not launch at all or rather has the malicious message lol.

To open 3.0 beta 2, go to system settings -> privacy and security -> scroll down and click allow. A simple google search would've fixed that issue for you.

**Comment by MEMORIEmusic** [comment]
Duplicate of #1108

**Comment by dzy1997** [comment]
> Duplicate of #1108

I have read about that issue, the Playcover and macOS versions differ, so they might not be the same problem. Also that issue happened last September, while mine happened only after I updated macOS to 14.2.1 yesterday.

**Comment by MEMORIEmusic** [comment]
> > Duplicate of #1108
> 
> 
> 
> I have read about that issue, the Playcover and macOS versions differ, so they might not be the same problem. Also that issue happened last September, while mine happened only after I updated macOS to 14.2.1 yesterday.

The issue arises in macOS 14.1, and is same on all later versions. 
At least try PlayCover 3.0 beta 2

**Comment by s7v3n7** [comment]
Apologies, Correction. I know about the Open at Privacy part. When I mean it won't open I mean it won't let me open by showing the warning "Malicious" message. After, I click okay it will just crash.
I am on MacOS 14.2.1. I was able to open and install application with the 2.0.5 version. However, all the application will crash when launch.

After I have installed the Nightly version 607.
1 - "Move to Application Folder" Windows shows up
2 - Hit "Move to Application Folder" 
3 - The "Malicious" warning will show up 
4 - Hit okay 
5 - "Move to Application Folder" shows up again then crashes (its like a Loop)
6 - Open Playcover.app on Privacy Setting .... Nothing happens
7 - Error message "You can't open the application "PlayCover.app" because it may be damaged or incomplete."

2nd Attempt to reinstall, same effect with steps reproduced up to step 4
5 - An error Occurred Message, The file "Playcover.app" could not be opened because there is no such file. At the same time that I receive this error message, the Playcover.app file disappeared from Application.

3rd Attempt to reinstall
1 - settings > Misc > Change Application type public.app-catoegory.none to public.app-catogery.games

Looks like Playcover is still open without crashing or other messages.
Attempted to install application (Genshin Impact 4.3.0 and Tower of Fantasy 3.5.0):
1 - dragging the IPA file 
2 - an Error Occured at 80% during unwrapping.....The file "Playtools.framework" couldn't be opened because there is no such file.
3 - hit okay, "an Error Occured" with Playcover shows creating a wrapper
4 - Application will not open/nothing happens regardless of any settings ticked.

4th Attempt but this time with 3.0 Beta 2 Version. Previously installed application and settings remained.
Error messages pop out but it does not crash. This time for some reason the Application mysteriously launched. Currently have no issue with login and updating the latest packages.

**Comment by duraki** [comment]
Why is this closed?
